contents/docs/feature-flags/common-questions.mdx

```diff
@@ -44,7 +44,7 @@ Check if an ad-blocker is interfering with PostHog calls. If this is the case, y
 
 If your [release conditions](/docs/feature-flags/creating-feature-flags#release-conditions) depend on user or group properties, and you immediately evaluate the feature flag after updating them, the properties may not be ingested in time to calculate the correct flag value.
 
-In this case, when making your `getFeatureFlag()` call, you can [manually include in these properties](/docs/feature-flags/adding-feature-flag-code#advanced-overriding-server-properties) in the method arguments.
+In this case, when making your `getFeatureFlag()` call, you can [manually include these properties](/docs/api/post-only-endpoints#advanced-overriding-server-properties) in the method arguments.
```

Comment:
hmm, we still want to point to the SDK docs, not the pure API ones, on closer inspection the problem seems to be that https://posthog.com/docs/feature-flags/adding-feature-flag-code?tab=Web#advanced-overriding-server-properties this page doesn't have the override section for web (it exists for other SDKs)

The override section is here: https://posthog.com/docs/libraries/js#overriding-server-properties - just adding this component to the adding-feature-flag-code section should be enough here 👀